### PR TITLE
Bioregion 1

### PR DESCRIPTION
**Pull request by Brian393**
@edwardsharp 

Here is my latest. Curious what it looks like to you. My Atom editor now says "The repository at !#$@^ has too many changed files to display." 

If it's ready for me to pull, could you instruct me in the proper way to do that?

thanks, Brian

### CODE REVIEW

**Comment by edwardsharp** [comment]
oh cool, i’ll give this a closer look this evening!

**Comment by edwardsharp** [comment]
hi @Brian393, 

sooo, this pull request is from your `Bioregion-1` branch to my `master` branch which doesn't feel quite right. since you're doing most of the work on this & ecotopia.today is hosted from your repo's `gh-pages` branch you don't need to worry about pull requests to my repo's master branch (like we talked about). 

what i keep mentioning to you is to ask me to review your commits before you put them into YOUR `master` branch. please, **before anything else**, can you let me know if this makes sense??

otherwise i've added feedback on the applicable lines.

File: public/geojson/VanUnderlayer.geojson [hunk 1]
```diff
@@ -0,0 +1,8 @@
...
...[ -122.7039, 45.6106 ], [ -122.7032, 45.6106 ], [ -122.7025, 45.6105 ], [ -122.7022, 45.6104 ], [ -122.7018, 45.6103 ], [ -122.6922, 45.6072 ], [ -122.6941, 45.6041 ], [ -122.6941, 45.6038 ], [ -122.694, 45.6036 ], [ -122.6935, 45.6034 ], [ -122.6933, 45.6032 ], [ -122.6931, 45.6026 ], [ -122.6932, 45.602 ], [ -122.6934, 45.6004 ], [ -122.6925, 45.5999 ], [ -122.6923, 45.5998 ], [ -122.685, 45.5985 ], [ -122.6871, 45.5916 ], [ -122.6884, 45.5917 ], [ -122.6894, 45.5917 ], [ -122.6905, 45.5914 ], [ -122.6908, 45.5915 ], [ -122.6928, 45.5921 ], [ -122.6944, 45.5926 ], [ -122.6965, 45.5933 ], [ -122.701, 45.5947 ], [ -122.7017, 45.5949 ], [ -122.7022, 45.5951 ], [ -122.7031, 45.5952 ], [ -122.7036, 45.5952 ], [ -122.7038, 45.5954 ], [ -122.7039, 45.5955 ], [ -122.7041, 45.5962 ], [ -122.7041, 45.5964 ], [ -122.7042, 45.5965 ], [ -122.7044, 45.5966 ], [ -122.7045, 45.5966 ], [ -122.7049, 45.5966 ], [ -122.7055, 45.5965 ], [ -122.7064, 45.5965 ], [ -122.7073, 45.5965 ], [ -122.7081, 45.5966 ], [ -122.7089, 45.5967 ], [ -122.7097, 45.5969 ], [ -122.7107, 45.5972 ], [ -122.711, 45.5973 ], [ -122.7112, 45.5973 ], [ -122.7116, 45.5974 ], [ -122.712, 45.5975 ], [ -122.7123, 45.5976 ] ] ] } }
```
Comment (edwardsharp) [inline public/geojson/VanUnderlayer.geojson:6]: why such long lines? your text editor can pretty print format json 😸

File: src/components/MapMegaregion.vue [hunk 2]
```diff
@@ -161,6 +164,55 @@ export default {
...
+                url: 'http://ecotopia.today/cascadia/Tiles/Vanport/{z}/{x}/{y}.png'
+              }),
+              opacity: 1,
+              minZoom: 7,
+              maxZoom: 20
+            })
+          ]
+        },
+
+// end work zone
```
Comment (edwardsharp) [inline src/components/MapMegaregion.vue]: these `work zone` & `end work zone` comments are a bit superfluous

File: src/views/watershed/WatershedHanfordPlumes.vue [hunk 3]
```diff
@@ -1,43 +1,87 @@
...
-      </p>
+    <div class="title">
+      <h1>Body time</h1>
     </div>
-  </div>
-</template>
+    <div class="video">
+      <iframe width="350" height="197" src="https://www.youtube.com/embed/Sp_FVqKjZQQ?rel=0&amp;start=349;autoplay=0&mute=0&amp;showinfo=0" frameborder="0" allow="autoplay; encrypted-media" allowfullscreen></iframe>
+    </div>
+    <div class="caption">Testimony of Tom Bailie and Jay Mullen<br>recorded by <a href="https://www.youtube.com/watch?v=Sp_FVqKjZQQ" target="_blank">Paul DeRienzo</a></div>
```
Comment (edwardsharp) [inline src/views/watershed/WatershedHanfordPlumes.vue]: you should use the attribute/value `rel="nofollow"` whenever you use `target="_blank"`.

**Comment by Brian393** [comment]
Hi ed, thanks, I will have a look at the comments as soon as possible (I'm
in Salina KS installing a show).

I do understand you would like to see the commits before I integrate them
to the master, that's great. How should I inform you when it's ready to be
reviewed? Let me know and I'll do it that way.

Best, Brian

**Comment by edwardsharp** [comment]
@Brian393 there's a variety of ways to inform me:

1. assign the pull request to me 🥇 
2. make a comment and `@` me 👍 
3. `@` me in the body of the pull request 👌 
4. email 😐 
5. ...smoke signals? 😓

**Pull request closed without merging**